components/lib/article-credits.php

```diff
@@ -0,0 +1,59 @@
+<?php
+function article_credits()
```

Comment:
Let's have post id as an argument to make sure we get it inside and outside of the wordpress loop.